src/Logger.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Component;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger as MonologLogger;
+
+class Logger extends MonologLogger
+{
+    public function __construct(
+        ?StreamHandler $errorHandler,
+        ?StreamHandler $logHandler
```

Comment:
ja nevim, mozna bych to zjednodussil jen na handlers?

Reply:
No, ale to by mi pak nefungovalo to, že když je to NULL, tak si ty dva udělám sám.

Reply:
testoval bys jestli je to prazdny pole, kdyz tam budes mit gelf logger, tak ti staci jeden handler afaik

Reply:
OK, done